app.py

```diff
@@ -1,28 +1,26 @@
+import random
```

Comment:
This is going to sound dumb but, again, developers will expect it. Imports should always be listed in this order.
```py
# All built in modules
import os

# All installed modules
from flask import Flask
import requests
from sqlalchemy.exc import IntegrityError

# All core/global/shared file for your application
from forms import SignupForm
from Database import connect_db, signup

# Any imports that are specific to this module -- you don't have any here but for example
# in a service you may have a helper class or module of functions for that is just for
# that service. Maybe...
from data_transformers import transform_player_stat_response
```